data/telegraf_plugins.yml

```diff
@@ -2166,6 +2166,13 @@ processor:
     introduced: 1.14.0
     tags: [linux, macos, windows]
 
+  - name: Starlark
+    id: starlark
+    description: |
+      The Starlark processor plugin calls a Starlark function for each matched metric, allowing for custom programmatic metric processing.
```

Comment:
maybe under `Starklark function` we can also link to the [starlark repo specs](https://github.com/google/starlark-go/blob/master/doc/spec.md)

Reply:
Updated!